docs/webserver.md

```diff
@@ -0,0 +1,39 @@
+# Webserver
+
+| | web02.rackspace.theforeman.org |
+| - | - |
+| type | OpenStack VM |
+| OS | CentOS 7 |
+| CPUs | 2 |
+| RAM | 2GB |
+| Storage | /dev/xvda (40GB): root, /dev/xvdb (100GB) + /dev/xvdc (50GB): data |
+
+## Domains
+
+These domains are all hosted on the webserver.
+
+* theforeman.org, www.theforeman.org
+* deb.theforeman.org
+* debugs.theforeman.org
+* downloads.theforeman.org
+* stagingdeb.theforeman.org
+* yum.theforeman.org
+
+## Volumes
+
+/var/www is mounted on a separate 140GB block device via LVM.  /var/www/freight* contain the staging areas for freight (deb), and /var/www/vhosts contain the web roots themselves.
+
+## Firewall
+
+firewalld is configured with TCP ports:
+
+* 22
+* 80
+* 443
+* 873 open
+
+## Other
+
+* freight and freightstage users have private auto-signing GPG key imported
+
+In case of maintenance, a template page and config file snippet are under /var/www/503.  The config should be copied into each vhost.
```

Comment:
I'm not sure this still works with Fastly as a CDN in front.

Reply:
If the backend serves 50x, Fastly will serve old content.